black.py

```diff
@@ -96,7 +120,7 @@ class InvalidInput(ValueError):
     """Raised when input source code fails all parse attempts."""
 
 
-class WriteBack(Enum):
+class WriteBack(IntEnum):
```

Comment:
What is the motivation for this change? I generally prefer Enum to IntEnum where there are no compatibility constraints.

Reply:
It is to avoid explicit conversion if `TargetVersion.value` to `int`. For `enum.Enum` `value` could be `Any`, so `mypy` complains if you use it as `int`. As you see, I use `value` in `TargetValue.get_sys_version`. `IntEnum`  is a very simple class, equivalet to `class TargetVersion(int, enum.Enum`): ...